uc-crux-llvm/src/UCCrux/LLVM/Classify.hs

```diff
@@ -41,6 +41,7 @@ import qualified Data.Set as Set
 import           Data.Maybe (maybeToList)
 import           Data.Text (Text)
 import qualified Data.Text as Text
+import           GHC.Natural (Natural)
```

Comment:
This is somewhat pedantic, but I tend to prefer `Numeric.Natural` (http://hackage.haskell.org/package/base-4.15.0.0/docs/Numeric-Natural.html) to avoid feeling bad about depending on GHC internals

Reply:
> to avoid feeling bad about depending on GHC internals

I feel like this ship has sailed... :) At the top of this file:
```haskell
{-# LANGUAGE DataKinds #-}
{-# LANGUAGE FlexibleContexts #-}
{-# LANGUAGE GADTs #-}
{-# LANGUAGE LambdaCase #-}
{-# LANGUAGE OverloadedStrings #-}
{-# LANGUAGE PolyKinds #-}
{-# LANGUAGE RankNTypes #-}
{-# LANGUAGE ScopedTypeVariables #-}
{-# LANGUAGE TypeFamilies #-}
{-# LANGUAGE TypeOperators #-}
{-# LANGUAGE ViewPatterns #-}
```
It makes no difference to me though, I'm happy to change it and use `Numeric` in the future.